Lib/_pyrepl/readline.py

```diff
@@ -161,6 +169,11 @@ def get_completions(self, stem: str) -> list[str]:
             result.sort()
         return result
 
+    def get_module_completions(self) -> list[str]:
+        completer = ModuleCompleter(namespace={'__package__': '_pyrepl'})  # TODO: namespace?
+        line = self.get_line()
+        return completer.get_completions(line)
```

Comment:
Can this raise? What do we want to do if any of the pkgutil raises for any reason?

Reply:
It shouldn't unless `iter_modules` or `find_spec` raise. Since those can call code from user-supplied finders, they can raise an arbitrary exception. I could add a simple `except Exception` block and simply return no completions if that happens. To the user it would look like no completions are available rather than outright crashing the repl.

Reply:
Added in [fd81999](https://github.com/python/cpython/pull/129329/commits/fd81999df165f15a901cccef6e6b0ca35b4b2324)